tmt/base.py

```diff
@@ -2862,23 +2903,30 @@ def tests(
             conditions: Optional[list[str]] = None,
             unique: bool = True,
             links: Optional[list['LinkNeedle']] = None,
-            excludes: Optional[list[str]] = None
+            excludes: Optional[list[str]] = None,
+            apply_command_line: Optional[bool] = True
```

Comment:
If the parameter is only a boolean flag, and it never can be `None`, `Optional` is not needed. Check `unique` few lines above, the same case.